torchaudio/models/wavernn.py

```diff
@@ -324,3 +349,27 @@ def forward(self, waveform: Tensor, specgram: Tensor) -> Tensor:
 
         # bring back channel dimension
         return x.unsqueeze(1)
+
+
+def get_pretrained_wavernn(checkpoint_name: str, progress: bool = True) -> WaveRNN:
+    r"""Get pretrained WaveRNN model.
+
+    Here are the available checkpoints:
+
+    - wavernn_10k_epochs_8bits_ljspeech
```

Comment:
Why not put this under `checkpoint_name` section? I think that is more suitable place.

https://262737-90321822-gh.circle-artifacts.com/0/docs/models.html#get-pretrained-wavernn

Reply:
I've moved it under the checkpoint_name [here](https://github.com/pytorch/audio/pull/1612/files#diff-7f4ad259da182d3840fe4c75dd0c0b7b9fe30f31503df0eff87b1e5c9378cac4R358)